homeassistant/components/homekit/type_sensors.py

```diff
@@ -32,7 +31,7 @@ def __init__(self, hass, entity_id, name, **kwargs):
 
         serv_temp = add_preload_service(self, SERV_TEMPERATURE_SENSOR)
         self.char_temp = serv_temp.get_characteristic(CHAR_CURRENT_TEMPERATURE)
-        override_properties(self.char_temp, PROP_CELSIUS)
+        self.char_temp.override_properties(properties=PROP_CELSIUS)
```

Comment:
What is this for? As far I can tell your overwriting the char `max` and `min` numbers with almost identical values (`min=-273;max=999` for [HA](https://github.com/cdce8p/home-assistant/blob/6f181779f701bcd3217fd0947c6677e7b77010ba/homeassistant/components/homekit/const.py#L72) vs `min=-273.1;max=1000` for [HAP](https://github.com/ikalchev/HAP-python/blob/dev/pyhap/resources/characteristics.json#L364))

Reply:
I'm not sure if it was the right move to override this in `HAP-python`. In case we change that in the future, this should prevent any issues.